static/js/index.js

```diff
@@ -0,0 +1,167 @@
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project3',
+  },
+  {
+    id: 4,
+    title: 'Uber',
+    list: {
+      list1: 'Uber',
+      list2: 'Lead Developer',
+      list3: '2015',
+    },
+    image: 'static/imgs/two.png',
+    desc: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project4',
+  },
+];
+// eslint-disable-next-line no-plusplus
+
+// ==========================================================================
+const burger = document.querySelector(
+  '.header__container__parhead__nav__ul__responsive',
+);
+const iwq = document.querySelector('.iwq');
+const parpopup = document.querySelector('.parpopup');
+// eslint-disable-next-line camelcase
+const parpopup__popup = document.querySelector('.parpopup__popup');
+const navLinks = document.querySelectorAll('.mobile-nav-links');
+const email = document.querySelector('.form__control > input#email');
+const form = document.querySelector('form');
+const emailerror = document.querySelector('#emailerror');
+// ===========================================================================
+burger.addEventListener('click', () => {
+  if (burger.classList.contains('change')) {
+    burger.classList.remove('change');
+  } else {
+    burger.classList.add('change');
+  }
+
+  if (iwq.classList.contains('show')) {
+    iwq.classList.remove('show');
+  } else {
+    iwq.classList.add('show');
+  }
+});
+// ============================================================================
+// eslint-disable-next-line no-unused-vars
+function detail(c) {
+  parpopup.style.display = 'flex';
+  const update = details.filter((item) => item.category === c);
+  parpopup__popup.innerHTML = `
+  <div class="head">
+  <span class="name">${update[0].title}</span>
+  <span id="close" onclick="closes()">&times;</span>
+</div>
+<ul>
+  <li><a href="">${update[0].list.list1}</a></li>
+  <li><a href="">${update[0].list.list2}</a></li>
+  <li><a href="">${update[0].list.list3}</a></li>
+</ul>
+<div class="popup__responsive">
+  <img src="${update[0].image}" alt="" class="responsive" />
+</div>
+<div class="types">
+  <div class="type">
+    <p>${update[0].desc}</p>
+  </div>
+  <div class="type">
+    <ul>
+      <li>${update[0].lang.lang1}</li>
+      <li>${update[0].lang.lang2}</li>
+      <li>${update[0].lang.lang3}</li>
+      <li>${update[0].lang.lang4}</li>
+    </ul>
+    <hr />
+    <div class="btns live">
+      <button class="live"><span>See live</span> <img src="static/imgs/Icon.png"/></button><button class="live">See source <i class="fab fa-github"></i></button>
+    </div>
+  </div>
+</div>
+  
+  
+  `;
+}
+// ========================================================================
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: